app/images/mask-icon.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 46 (44423) - http://www.bohemiancoding.com/sketch -->
+    <title>mask-icon</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
If it's not against the T&C for Sketch, I'd rather remove the description here and generator comment above.